client/app/telecom/telephony/alias/configuration/mode/telecom-telephony-alias-configuration-mode.html

```diff
@@ -7,12 +7,12 @@
         <h1 data-translate="telephony_alias_configuration_mode_title"></h1>
```

Comment:
This whole file will be rewrite in a later feature, I've just changed the controller name.